cmd/gonomics/usage.go

```diff
@@ -4,12 +4,10 @@
 package main
 
 import (
-	"bufio"
-	"bytes"
 	"fmt"
 	"github.com/vertgenlab/gonomics/exception"
 	"github.com/vertgenlab/gonomics/fileio"
-	"io/ioutil"
+	"golang.org/x/exp/slices"
```

Comment:
Do we need to update go.mod/num with this?

Reply:
We already require x/exp so no update needed.